### PR TITLE
Chore/update gradio template

### LINKED ISSUE #42: fix: Gradio app template
need to modify Gradio app template according to the changes in data from `sidewalks` to `PETS`.

### PR DESCRIPTION
**Pull request by deep-diver**
Closes #42

### CODE REVIEW

File: training_pipeline/apps/gradio/semantic_segmentation/app.py [hunk 1]
```diff
@@ -88,7 +86,7 @@ def get_predictions(image: Image):
     allow_flagging="never",
     title=title,
     description=description,
-    examples=[["test-image.jpg"]],
+    examples=[["test-image1.png"], ["test-image2.png"]],
```
Comment (sayakpaul) [inline training_pipeline/apps/gradio/semantic_segmentation/app.py:89]: Cute animals. Can we a few more cute pics of dogs and cats? :D

File: training_pipeline/apps/gradio/semantic_segmentation/palette.txt [hunk 2]
```diff
@@ -1,35 +1,3 @@
...
-[170, 84, 127]
-[170, 170, 127]
-[170, 255, 127]
-[255, 0, 127]
-[255, 84, 127]
-[255, 170, 127]
\ No newline at end of file
+[0, 10, 146]
+[38, 0, 44]
+[255, 232, 0]
```
Comment (sayakpaul) [inline training_pipeline/apps/gradio/semantic_segmentation/palette.txt:3]: Short and simple. 

Maybe add a comment on top of this? 

* 0 for background 
* 1 for dog
* 2 for cat

**Review by sayakpaul — APPROVED** [review]
LGTM, just minor comment.

**Comment by deep-diver** [comment]
@sayakpaul 

added more example photos from TensorFlow official tutorial, 
added comments oh palette.txt
- actually, this tutorial classifies inner, outer, border of an object

**Pull request merged**